changelog_unreleased/scss/4698.md

```diff
@@ -0,0 +1,19 @@
+#### Add space before unary minus followed by a function call (#4698 by @pamelalozano)
```

Comment:
Please specify the PR number here and in the file name, not the issue number

Reply:
Got it, done!